xc/xenops_server_xen.ml

```diff
@@ -958,18 +947,29 @@ module VM = struct
                        ~platformdata:vm.Xenops_interface.Vm.platformdata
```

Comment:
here as well, we could use `default_persistent_t with ...` and list only the updated fields

Reply:
Fixed.